concordia/tests/test_views.py

```diff
@@ -443,8 +442,8 @@ def test_asset_reservation_tombstone(self):
         """
         asset = create_asset()
         self.login_user()
-
-        reservation_token = token_hex(25)
+        fake_session = dict()
```

Comment:
We could use a `RequestFactor` to generate these and avoid needing to change the `get_or_create_reservation` signature: https://docs.djangoproject.com/en/3.0/topics/testing/advanced/#the-request-factory